composer.json

```diff
@@ -39,7 +39,8 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.3",
+		"php": ">=5.4",
+		"phpbb/phpbb": ">=3.2.0@dev",
```

Comment:
@michaelcullum [some extensions have dependencies](https://github.com/board3/Board3-Portal/blob/2.1.x/composer.json#L23) (and of course our extensions like to have their composer.lock files updated from time to time).